src/libUtils/SysCommand.h

```diff
@@ -17,17 +17,47 @@
 #ifndef __SYSCOMMAND_H__
 #define __SYSCOMMAND_H__
 
+#include <array>
 #include <cstdlib>
 #include <fstream>
 #include <iostream>
+#include <memory>
 #include <string>
 
-void ExecuteCmd(const std::string cmd)
+const std::string EXEC_CMD_LOG = "ExecuteCmd.txt";
+
+class SysCommand
 {
-    int ret = std::system((cmd + " > ExecuteCmd.txt").c_str());
-    (void)ret;
-    LOG_GENERAL(INFO,
-                "ExecuteCmd: " << std::ifstream("ExecuteCmd.txt").rdbuf());
-}
+public:
+    static void ExecuteCmd(const std::string cmd)
+    {
+        int ret = std::system((cmd + " > " + EXEC_CMD_LOG).c_str());
```

Comment:
I don't see why only keeps stdout but leaves stderr. If it's for debugging purpose, then both can be kept in a temporary log file.